client/scripts/templates/template.min.js

```diff
@@ -0,0 +1,254 @@
+(function() {
```

Comment:
This file shouldn't be committed to source control.  I'll add it to the .gitignore so it doesn't accidentally get re-added in a future PR.